charts/openstack-cloud-controller-manager/templates/daemonset.yaml

```diff
@@ -79,6 +80,10 @@ spec:
           env:
             - name: CLOUD_CONFIG
               value: /etc/config/cloud.conf
+            - name: CLUSTER_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
```

Comment:
I don't think using nodeName is a good idea, maybe just set a default value `kubernetes`?